includes/helpers/class-authenticated-listing-view.php

```diff
@@ -2,6 +2,35 @@
 
 class WPBDP__Authenticated_Listing_View extends WPBDP__View {
 
+    /**
+     * Load resources required for the view
+     *
+     * @since x.x
+     */
+    public function enqueue_resources() {
+
+        // CSS used for plan buttons on the listing page only.
+        $custom_css = "
+		.wpbdp-plan-info-box .wpbdp-plan-price input[type=radio]+ label span:before{
+			content: '" . esc_attr__( 'Select', 'business-directory-plugin' ) . "';
+		}
+		.wpbdp-plan-info-box .wpbdp-plan-price input[type=radio]:checked + label span:before{
+			content: '" . esc_attr__( 'Selected', 'business-directory-plugin' ) . "';
+		}";
+		wp_add_inline_style( 'wpbdp-base-css', $custom_css );
+
+        $this->enqueue_custom_resources();
+    }
+
+    /**
+     * Load custom resources in classes that extend this class
+     *
+     * @since x.x
+     */
+    public function enqueue_custom_resources() {
```

Comment:
Empty function here

Reply:
Explained below

Reply:
There should be a comment here for the empty function. Also, be sure to use tabs instead of spaces.

Reply:
Changes uploaded. Fixed space indentation in the rest of the file as well

Reply:
For empty I have seen above is only if there are params like https://github.com/WordPress/WordPress/blob/4152fbfdb2e41a5de19b664a666e6620f13499b4/wp-admin/includes/class-wp-list-table.php#L1403 but most common is having the comment in the empty function https://wordpress.stackexchange.com/questions/145569/actions-functions-and-conditionals